### PR TITLE
chore(sycamore-template): update sycamore to v0.9

### PR DESCRIPTION
**Pull request by lukechu10**
Updates the Sycamore template to the new v0.9 version. Also updates the dependency on `serde-wasm-bidngen`.

### CODE REVIEW

File: templates/template-sycamore/src/app.rs.lte [hunk 1]
```diff
@@ -16,46 +16,49 @@ struct GreetArgs<'a> {
...
 
-    let greet = move |e: Event| {
+    let greet = move |e: SubmitEvent| {
         e.prevent_default();
-        spawn_local_scoped(cx, async move {
+        spawn_local_scoped(async move {
             // Learn more about Tauri commands at {% if v2 %}https://tauri.app/develop/calling-rust/{% else %}https://v1.tauri.app/v1/guides/features/command{% endif %}
-            let args = serde_wasm_bindgen::to_value(&GreetArgs { name: &name.get() }).unwrap();
+            let args = serde_wasm_bindgen::to_value(&GreetArgs {
+				name: &name.get()
```
Comment (amrbashir) [inline templates/template-sycamore/src/app.rs.lte]: looks like in the new version, `.get` requires `Copy` trait on the signal value, which seems weird but looking over the docs we need to use either `.with` or `.with_untracked`

**Review by amrbashir — CHANGES_REQUESTED** [review]

**Review by amrbashir — DISMISSED** [review]
Thank you

File: templates/template-sycamore/src/app.rs.lte (hunk 1 above)
Comment (lukechu10) [reply templates/template-sycamore/src/app.rs.lte]: Sorry forgot about that. Should be fixed now.

File: .changes/update-sycamore.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"create-tauri-app": patch
```
Comment (amrbashir) [inline .changes/update-sycamore.md:2]:
```suggestion
"create-tauri-app": patch
"create-tauri-app-js": patch
```

**Review by amrbashir — APPROVED** [review]

**Pull request merged**